docs/installation.md

```diff
@@ -5,19 +5,27 @@ This is the installation instructions for the Zonemaster CLI utility.
 The documentation covers the following operating systems:
 
  * [1] <a href="#Debian">Ubuntu 12.04 (LTS)</a>
- * [2] <a href="#Debian">Ubuntu 14.04 (LTS)</a>
- * [3] <a href="#Debian">Debian Wheezy (version 7)</a>
- * [4] <a href="#FreeBSD">FreeBSD 10.1</a>
- * [5] <a href="#CentOS">CentOS 7 </a>
+ * [2] <a href="#Debian">Debian Jessie (version 8) - 64 bits</a>
+ * [3] <a href="#FreeBSD">FreeBSD 10.1</a>
+ * [4] <a href="#CentOS">CentOS 7 </a>
+
+>
+> Note: We assume the installation instructions will work for earlier OS
+> versions too. If you have any issue in installing the Zonemaster engine with
+> earlier versions, please send a mail with details to contact@zonemaster.net
+>
+
 
 ## Prerequisite
 
 Before you install the Zonemaster CLI utility, you need the
 Zonemaster Engine test framework installed. Please see the
 [Zonemaster Engine installation instructions](https://github.com/dotse/zonemaster-engine/blob/master/docs/installation.md).
 
```

Comment:
As mentioned earlier, cpan or cpanm is not a prerequisite, as it would have been already installed for the engine.